habana/1.10.0/ubi8-python-3.8/Pipfile

```diff
@@ -33,7 +33,7 @@ pybind11 = "==2.6.2"
 elyra-pipeline-editor-extension = "~=3.15.0"
 elyra-python-editor-extension = "~=3.15.0"
 elyra-code-snippet-extension = "~=3.15.0"
-kfp-tekton = "<1.6.0"
+kfp-tekton = "==1.5.7"
```

Comment:
Hm, I was able to find some info WRT [1.6.0 release](https://github.com/kubeflow/kfp-tekton/releases/tag/v1.6.0) but nothing about the 1.5.7 to check the differences - not even a branch to compare against :face_with_diagonal_mouth:

Reply:
checking the lock file, version of kfp-tekton in there has not changed, so the image was clearly using 1.5.7 before anyways, and this is not a functional change

Reply:
Ah, good spot, Jiri. You are right. Thanks! :slightly_smiling_face: